src/components/OfferCard/components/InclusionsGroup/InclusionsGroup.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, FeatureListItem, NakedButton, Text } from 'components';
+
+const InclusionGroup = ({
+  inclusionsLabel,
+  inclusionsCallToActionLabel,
+  onClickInclusionsCallToAction,
+  ...inclusionsProps
```

Comment:
```suggestion
  inclusions
```
Can destructure `inclusions` prop as well?